tesseract_core/include/tesseract_core/basic_types.h

```diff
@@ -34,13 +34,15 @@
 #include <vector>
 #include <memory>
 #include <functional>
-#include <eigen_stl_containers/eigen_stl_containers.h>
+#include <map>
+
 
 namespace tesseract
 {
 typedef Eigen::Matrix<double, Eigen::Dynamic, Eigen::Dynamic, Eigen::RowMajor> TrajArray;
-typedef std::vector<Eigen::Affine3d, Eigen::aligned_allocator<Eigen::Affine3d> > vector_Affine3d;
-typedef std::map<std::string, Eigen::Affine3d> TransformMap;
+typedef std::vector<Eigen::Isometry3d, Eigen::aligned_allocator<Eigen::Isometry3d> > VectorIsometry3d;
+typedef std::vector<Eigen::Vector4d, Eigen::aligned_allocator<Eigen::Vector4d> > VectorVector4d;
+typedef std::map<std::string, Eigen::Isometry3d> TransformMap;
```

Comment:
Technically, `std::map` should also take an aligned allocator here. See Eigen [docs](https://eigen.tuxfamily.org/dox/group__TopicStlContainers.html).

Shameless plug, you can use templated aliases to make this code very concise and general. See my [PR](https://github.com/ros/eigen_stl_containers/pull/14). For your convenience:
```c++
template <typename Key, typename Value>
using AlignedMap = std::map<Key, Value, std::less<Key>, Eigen::aligned_allocator<std::pair<const Key, Value>>>; 
```
```c++
template <typename T>
using AlignedVector = std::vector<T, Eigen::aligned_allocator<T>>;
```